pages/use-solid/apps.md

```diff
@@ -70,11 +70,12 @@ redirect_from:
 | [RiseFor Mobilisation](https://git.happy-dev.fr/startinblox/applications/risefor-mobilisation) <br>(deployed as [Referendum Signons](https://referendum.signons.fr)) | An app for organising referenda; it uses Solid under the hood. |
 | [Solidbase](https://app.solidbase.info) | Agricultural project management. <br>- [Source code](https://lab.allmende.io/solidbase/solidbase) is under [GNU AGPL3 (c) 2017](https://lab.allmende.io/solidbase/solidbase/blob/master/LICENSE) [Allmende Lab](https://lab.allmende.io) |
 
-## Blogs
+## Content, Notes, Blogging and Publishing
 
 | Application                                                  | Description                                                                                               |
 |--------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------|
-| [Concept](https://useconcept.art) ([GitHub](https://github.com/travis/concept)) |  An embarrassingly simple clone of [notion.so](https://notion.so) built on Solid, currently in early alpha.     |
+| [Concept](https://useconcept.art) ([GitHub](https://github.com/travis/concept)) |  An embarrassingly simple clone of [notion.so](https://notion.so) built on Solid. Deprecated by Understory Garden.    |
```

Comment:
Let's just remove this if it no longer works and the replacement is already listed.

```suggestion
```